zenith_utils/src/postgres_backend.rs

```diff
@@ -76,19 +79,48 @@ pub enum ProcessMsgResult {
     Break,
 }
 
+pub enum Stream {
```

Comment:
It'd be nice to have a brief comment explaining why we need this (and perhaps what it is, though that's easier to understand from viewing it.)